### ISSUE TITLE
Accordion: accessibility violation.

### ISSUE THREAD

**Issue by brunano21** [ibm-js/deliteful#646]
At this stage, each header has `aria-controls` set even though its panel is not visible.
This represents an accessibility violation, as stated below:

> **Elements that use WAI-ARIA properties must reference valid IDs**
> Elements that use WAI-ARIA properties must reference valid IDs to enable compatibility with assistive technologies.
> Following is a list of invalid ID references:
> 
>    1. Element referenced by ID doesn't exist.
>    2. WAI-ARIA properties only supports single ID reference but multiple provided.
>    3. Empty or whitespace ID reference.
>    4. _Element referenced by ID is hidden._ 

`aria-controls` attribute should be set only when the panel is visible and should be removed once the panel is hidden.
